web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -1538,8 +1543,40 @@
               });
             }
             return defer.promise;
-          }
+          },
+
+          /**
+           * Check for online resource that could be bound to the layer.
+           * WPS, downloads, WFS etc..
+           *
+           * @param {ol.Layer} layer
+           * @param {string} linkGroup
+           */
+          feedLayerWithRelated: function(layer, linkGroup) {
+            var md = layer.get('md');
 
+            if (!linkGroup) {
+              console.warn('The layer has not been found in any group: ' +
+                  layer.getSource().getParams().LAYERS);
+              return;
+            }
+
+            // We can bind layer and download/process
+            if (md.getLinksByType(linkGroup, '#OGC:WMTS',
```

Comment:
Can be used  `searchSettings.linkTypes`: https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/views/default/config.js#L206 instead of hardcoding the protocols?